docs/source/ko/_toctree.yml

```
@@ -339,6 +339,10 @@
         title: (번역중) LED
       - local: model_doc/llama
         title: LLaMA
+      - local: model_doc/llama2
+        title: LLaMA2
+      - local: model_doc/llama
```

```suggestion
      - local: model_doc/llama
```
위와 중복되는 라인이라 삭제가 필요합니다!